### PR TITLE
DOC: Updating the API documentation

### PR DESCRIPTION
**Pull request by greglucas**
Updating the documentation to be consistent with the current workflow. Also giving a description of a typical frontend call sequence.

### CODE REVIEW

**Comment by greglucas** [comment]
ping @jennyknuth for a review on this one to make sure it is what you are expecting with respect to the API endpoints.

File: README.md [hunk 1]
```diff
@@ -2,25 +2,48 @@
 
 Version 0.1
 
-Overview of the input/output requirements for VECTOR.
+Overview of the input/output requirements for VECTOR. The typical frontend
+workflow will be (1) a call to the `/geometry` endpoint to get a unique `<id>`,
+followed by (2) a call to the `/singlepoint/<id>` endpoint with the desired
```
Comment (jennyknuth) [inline README.md]: I think the standard notation for a variable in an endpoint path is `/singlepoint/:id`. But your way works too!

File: README.md [hunk 2]
```diff
@@ -2,25 +2,48 @@
...
+`/image/<id>` endpoint will return an image of the satellite geometry that was
+used in the computations.
 
 ## Endpoints
 
-- /api/image : POST
+- /geometry : POST
 
-This endpoint accepts a VRML `.wrl` file for upload and returns a `.png` image of the file.
+    This endpoint is for uploading fils and accepts a VRML `.wrl` file.
```
Comment (jennyknuth) [inline README.md]: files

File: README.md [hunk 3]
```diff
@@ -2,25 +2,48 @@
...
+
+    This endpoint says we want the requested satellite's geometry file that we
+    already have locally, so there is no need to upload a file. This endpoint
+    will return a unique `<id>` to be associated with the requested satellite.
+
+    The named satellites available are:
+
+    - SORCE
+    - CubeSat, 1U
+    - CubeSat, 3U with solar panels
```
Comment (jennyknuth) [inline README.md]: Did you leave out the possible values for sat_name (their machine readable values) on purpose?

File: README.md [hunk 4]
```diff
@@ -2,25 +2,48 @@
...
+
+- /singlepoint/<id> : POST
+    Receives the json data from the frontend website and makes
```
Comment (jennyknuth) [inline README.md]: JSON? since you did VRML…

Comment (jennyknuth) [reply README.md]: also below…

File: README.md [hunk 5]
```diff
@@ -85,22 +108,22 @@ energyAccomodation: float
...
+"speed": 7800.45,
 "composition": {"O": 1e11,
                 "O2": 1e6,
                 "N2": 1e6,
                 "He": 1e6,
                 "H": 1e4},
 "accommodationModel": "SESAM",
-"energyAccomodation": 0,
-"surfaceMass": 0}
+"energyAccomodation": 0.93,
```
Comment (jennyknuth) [inline README.md]: Accommodation (all the mm's)

Comment (jennyknuth) [reply README.md]: elsewhere as well…

**Review by jennyknuth — COMMENTED** [review]
Sorry it took me so long to realize this was here!

One other possible thing to add are the following requirements for inputs: 
'diameter must be positive'
'length must be positive'
'area must be positive'
'pitch angle must be between -90 and 90'
'sideslip angle must be between -180 and 180'
'temperature must be positive'
'speed must be positive'
 'composition must be positive'
 'energy accommodation must be between 0 and 1'
 'surface mass must be 1 or greater'

File: README.md (hunk 5 above)
Comment (greglucas) [reply README.md]: MMmmm.. I must not have been hungry while writing :)

File: README.md (hunk 3 above)
Comment (greglucas) [reply README.md]: This was before we moved it over to the API specific repo. Thanks for catching that!

File: README.md (hunk 1 above)
Comment (greglucas) [reply README.md]: Thanks, I think I see your way more frequently, so I changed it.

**Comment by greglucas** [comment]
Thanks @jennyknuth! I just pushed up the suggested changes.

File: README.md [hunk 6]
```diff
@@ -2,60 +2,81 @@
...
-The backend will then parse the payload and run VECTOR with the proper inputs to get the return values.
-The return from the VECTOR code will then be put in the response payload to send back to the frontend.
+    This endpoint says we want the requested satellite's geometry file that we
+    already have locally, so there is no need to upload a file. This endpoint
+    will return a unique `id` to be associated with the requested satellite.
+
+    The named satellites available are:
+  - `sorce`, The SORCE satellite
+  - `cubesat1u`, A 1U CubeSat
+  - `deployable3u`, A 3U CubeSat with solar panels
```
Comment (jennyknuth) [inline README.md]: totally knitpicky, but maybe we don't want the articles capitalized after the comma?

**Review by jennyknuth — APPROVED** [review]

File: README.md (hunk 6 above)
Comment (greglucas) [reply README.md]: Good to catch those things, because they wouldn't get addressed later I'm sure :) Thanks!

**Pull request merged**